tests/provisioner/main.tf

```diff
@@ -0,0 +1,5 @@
+resource "terraform_data" "delete-resource" {
+  provisioner "local-exec" {
+    command = "opslevel delete ${var.resource_type} ${var.resource_id}"
```

Comment:
Do we have to worry about this accidentally targeting the wrong account?

Reply:
I can't think of a situation where it would. This could also be said for testing these resources in general. For both it's up the tester to set the token for the correct test account

Reply:
👍